UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.IntegrationTests/RadioNavigationalWarning/RNWTestHelper.cs

```diff
@@ -0,0 +1,174 @@
+﻿using FakeItEasy;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using radioNavigationalWarningDto = UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.DTO;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using UKHO.MaritimeSafetyInformation.Common;
+using UKHO.MaritimeSafetyInformation.Common.Configuration;
+using UKHO.MaritimeSafetyInformation.Web.Services;
+
+namespace UKHO.MaritimeSafetyInformation.IntegrationTests.RadioNavigationalWarning
+{
+    public class RNWTestHelper
+    {
+        public readonly RadioNavigationalWarningsContext _fakeContext;
+        public IOptions<RadioNavigationalWarningConfiguration> _fakeRadioNavigationalWarningConfiguration;
+        public IHttpContextAccessor _fakeHttpContextAccessor;
+        public ILogger<RNWRepository> _fakeLoggerRnwRepository;
+        public ILogger<RNWService> _fakeLoggerRnwService;
+
+        public RNWTestHelper()
+        {
+            DbContextOptionsBuilder<RadioNavigationalWarningsContext> builder = new DbContextOptionsBuilder<RadioNavigationalWarningsContext>()
+                                                                    .UseInMemoryDatabase("msi-in-db");
+            _fakeContext = new RadioNavigationalWarningsContext(builder.Options);
+            _fakeRadioNavigationalWarningConfiguration = A.Fake<IOptions<RadioNavigationalWarningConfiguration>>();
+            _fakeHttpContextAccessor = A.Fake<IHttpContextAccessor>();
+            _fakeLoggerRnwRepository = A.Fake<ILogger<RNWRepository>>();
+            _fakeLoggerRnwService = A.Fake<ILogger<RNWService>>();
+            _fakeRadioNavigationalWarningConfiguration.Value.AdminListRecordPerPage = 20;
+        }
+
+        public async Task SeedRadioNavigationalWarnings(List<radioNavigationalWarningDto.RadioNavigationalWarning> radioNavigationalWarning)
+        {
+            _fakeContext.RadioNavigationalWarnings.AddRange(radioNavigationalWarning);
+            await _fakeContext.SaveChangesAsync();
+        }
+
+        public async Task SeedWarningType(List<radioNavigationalWarningDto.WarningType> warningTypes)
+        {
+            _fakeContext.WarningType.AddRange(warningTypes);
+            await _fakeContext.SaveChangesAsync();
+        }
+
+        public async Task DeSeedRadioNavigationalWarnings()
+        {
+            DbSet<Common.Models.RadioNavigationalWarning.DTO.RadioNavigationalWarning> warnings = _fakeContext.RadioNavigationalWarnings;
```

Comment:
Name can be simplified 
```suggestion
            DbSet<radioNavigationalWarningDto.RadioNavigationalWarning> warnings = _fakeContext.RadioNavigationalWarnings;
```

Reply:
Changes are done as requestee, Thank You.